networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc

```diff
@@ -16,6 +16,34 @@ Red Hat supports the Kubernetes NMState Operator in production environments on b
 
 Before you can use NMState with {product-title}, you must install the Kubernetes NMState Operator.
 
+[NOTE]
+====
+The Kubernetes NMState Operator updates the network configuration of the secondary NIC. It cannot update the network configuration of the primary NIC.
```

Comment:
Nit: This should probably be "a secondary NIC" instead of "the" because there could be more than one.